src/DRAMPower/DRAMPower/util/bus.h

```diff
@@ -31,70 +34,170 @@ struct bus_stats_t {
 		return *this;
 	};
 
+	bus_stats_t& operator*=(const uint64_t rhs) {
+		this->bit_changes *= rhs;
+		this->ones *= rhs;
+		this->zeroes *= rhs;
+		this->ones_to_zeroes *= rhs;
+		this->zeroes_to_ones *= rhs;
+		return *this;
+	};
+
 	friend bus_stats_t operator+(bus_stats_t lhs, const bus_stats_t& rhs) {
 		return lhs += rhs;
 	}
+
+	friend bus_stats_t operator*(bus_stats_t lhs, const uint64_t rhs) {
+		return lhs *= rhs;
+	}
+
+	friend bus_stats_t operator*(const uint64_t lhs, bus_stats_t rhs) {
+		return rhs *= lhs;
+	}
 };
 
-// TODO: Idle state einbauen wenn Kommando fertig (done?)
 class Bus {
+
+private:
+	enum class __BusInitPatternSpec
```

Comment:
Ich glaube __ sollte nur in der Standardbibliothek genutzt werden.

Reply:
Abgeändert zu BusInitPatternSpec_